activities.html

```diff
@@ -64,12 +28,11 @@
       <nav>
           <ul id="menu">
               <li><a href="index.html">Main Page</a></li>
-              <li><a href="activities.html">Activities</a>
+              <li><a href="activities.html">Activities</a></li>
               <li><a href="resources.html">Resources</a></li>
-              <li><a href="aboutus.html">About us</a></li>
+              <li><a href="aboutUs.html">About us</a></li>
```

Comment:
Sorry!!

Oversight on my part!!  Thanks for letting me know. I'll correct it and
resubmit the pull request tomorrow.

;)
On Dec 12, 2015 4:15 PM, "risachi" <notifications@github.com> wrote:

> In activities.html
> <https://github.com/risachi/travel-project/pull/56#discussion_r47438679>:
>
> >                <li><a href="resources.html">Resources</a></li>
> > -              <li><a href="aboutus.html">About us</a></li>
> > +              <li><a href="aboutUs.html">About us</a></li>
>
> Hey! this un-does my change that was just merged in: making all filenames
> lower-case. I think we should only use lower case because we're running
> into problems due to the different operating systems we're using. (Linux
> vs. OS X)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/risachi/travel-project/pull/56/files#r47438679>.
>